tests/ReaderTest.cc

```diff
@@ -36,10 +36,69 @@ using namespace pulsar;
 static std::string serviceUrl = "pulsar://localhost:6650";
 static const std::string adminUrl = "http://localhost:8080/";
 
-TEST(ReaderTest, testSimpleReader) {
+TEST(ReaderTest, testPartitionIndex) {
```

Comment:
Why did you move the `testPartitionIndex` here? Please avoid making these unrelated changes.